.deepsource.toml

```diff
@@ -1,6 +1,6 @@
 version = 1
 
-test_patterns = ["tests/test_*/py"]
+test_patterns = ["tests/test_*.py"]
 
 exclude_patterns = ["documents/knowledge/*.*"]
```

Comment:
exclude `tests/test_elements.py` as well?

Reply:
it ignores the files based on the pattern.